src/olympia/files/helpers.py

```diff
@@ -33,6 +33,21 @@
 LOCKED = 'locked'
 LOCKED_TIMEOUT = 60 * 5
 
+SYNTAX_HIGHLIGHTER_ALIAS_MAPPING = {
+    'xul': 'xml',
+    'rdf': 'xml',
+    'jsm': 'js',
+    'json': 'js',
```

Comment:
True, I'll look into that. Yeah, I didn't touch any of that yet, we might be able to clean this up even more. If that's alright for everyone I'd do that in a separate pull request though to keep this separated.